website/api/views.py

```
@@ -1,33 +1,43 @@
 from django.core.mail import send_mail
-from django.db.models import Count, Q, Sum
+from django.db.models import Count, F, Q, Sum
+from django.http import HttpResponse
 from django.template.loader import render_to_string
 from django.utils import timezone
 from django.utils.text import slugify
 from rest_framework import filters, status, viewsets
 from rest_framework.authentication import TokenAuthentication
+from rest_framework.decorators import action, api_view
+from rest_framework.exceptions import NotFound, ParseError
 from rest_framework.decorators import action
```

Ruff (0.14.5)

Redefinition of unused `action` from line 24

Remove definition: `action`

(F811)